src/sidebar/util/fetch-config.js

```diff
@@ -75,3 +97,87 @@ export function fetchConfig(appConfig, window_ = window) {
     return mergedConfig;
   });
 }
+
+/**
+ * Merge client configuration from h service with config fetched from
+ * an embedding frame synchronously.
+ *
+ * Use this method to retrieve the config synchronously from the embedding
+ * frame's query string. See tests for more details.
```

Comment:
In the common case, configuration is passed from the parent frame to the child via a hash fragment rather than query string. To avoid this comment getting out of date, you could just omit mentioning _how_ the configuration is passed here.

I think the comment is possibly confusing here because "embedding frame" sounds like it refers to the parent document, whereas it actually refers to the _sidebar frame_.